src/components/FormikAutocomplete.jsx

```diff
@@ -25,7 +36,23 @@ function FormikAutocomplete(props) {
         multiple
         filterOptions={filterOptions}
         value={values}
-        onChange={(e, val) => formik.setFieldValue(id, val)}
+        inputValue={inputValue}
+        onChange={(e, val) =>
+          onChange ? onChange(val) : formik.setFieldValue(id, val)
+        }
+        onInputChange={(event, newInputValue) => {
+          const options = newInputValue.split(',');
+
+          if (options.length > 1) {
+            const vals = values.concat(
+              options.map((x) => x.trim()).filter((x) => x)
+            );
+            setInputValue('');
+            return onChange ? onChange(vals) : formik.setFieldValue(id, vals);
```

Comment:
Just noticed this shows up twice. Instead of having this show up twice just do it once near the top:
```js
const onChange = props.onChange ?? (e, val) => formik.setFieldValue(id, val);
```

Reply:
change it a bit since it was throwing an error when deleting chips.